presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/LogsHandler.java

```diff
@@ -245,4 +293,17 @@ else if (nullEnd) {
             return s -> level.name().equals(s);
         }
     }
+
+    private static Response internalError(
```

Comment:
You could migrate these two methods to the utility as well. Do the logging in the utility is also acceptable